src/content/docs/linter/index.mdx

```diff
@@ -4,180 +4,205 @@ description: How to use the Biome linter.
 
+### Safe fixes
 
-## Recommended rules
+Safe fixes are guaranteed to not change the semantic of your code.
+They can be applied without explicit review.
 
-When the linter is enabled, it recommends a number of rules. Recommended rules will emit error diagnostics. Below the list of recommended rules:
+To apply _safe fixes_, use `--apply`:
 
-<RecommendedRules />
+<PackageManagerBiomeCommand command="lint --apply ./src" />
 
+### Unsafe fixes
 
-## Ignoring Code
+Unsafe fixes may change the semantic of your program.
+Therefore, it's advised to manually review the changes.
 
-There are times when a developer wants to ignore a lint rule for a specific line of the code.
-You can achieve this by adding a suppression comment above the line that emits the lint diagnostic.
+To apply _unsafe fixes_, use `--apply-unsafe`:
 
-Suppression comments have the following format:
+<PackageManagerBiomeCommand command="lint --apply-unsafe ./src" />
 
-```js
-// biome-ignore lint: <explanation>
-// biome-ignore lint/suspicious/noDebugger: <explanation>
-```
+Note that this command also applies safe fixes.
 
-Where
+### Rule pillars
 
-- `biome-ignore` is the start of a suppression comment;
-- `lint` suppresses the linter;
-- `/suspicious/noDebugger`: **optional**, group and name of the rule you want to suppress;
-- `<explanation>` explanation why the rule is disabled
+We believe that rules should be informative and explain to the user why a rule is triggered and tell the user what they should to do fix the error.
+A rule should follow these **pillars**:
 
-Here's an example:
+1. Explain to the user the error. Generally, this is the message of the diagnostic.
+2. Explain to the user **why** the error is triggered. Generally, this is implemented with an additional node.
+3. Tell the user what they should do. Generally, this is implemented using a code action.
+   If a code action is not applicable a note should tell the user what they should do to fix the error.
+
+If you think a rule doesn't follow these pillars,
+please [open an issue](https://github.com/biomejs/biome/issues/new?assignees=&labels=S-To+triage&projects=&template=01_bug.yml&title=%F0%9F%90%9B+%3CTITLE%3E).
 
-```ts
-// biome-ignore lint: reason
-debugger;
-// biome-ignore lint/suspicious/noDebugger: reason
-debugger;
-```
 
 ## Configuration
 
-### Enable a lint rule
+Recommended rules are enabled by default and emit diagnostics with the `error` severity.
+Rules that are not recommended are disabled by default, but they can be enabled via [configuration](/guides/how-biome-works#configuration).
 
-Recommended rules are enabled by default and emit diagnostics with the error severity.
-Rules that are not recommended are disabled by default, but they can be enabled via configuration.
-The diagnostics emitted by these rules are displayed with the warning severity in the documentation.
+A rule is enabled whether its severity is either `error` or `warn` and is disabled if its severity is `off`.
```

Comment:
In our configuration, we have `level`, but here you say, "Its severity is off." I believe this is confusing. I think we shouldn't say "severity", and just say "you can turn off a rule with `"off"`"